code/modules/mob/living/basic/space_fauna/netherworld/creature.dm

```diff
@@ -100,3 +102,16 @@
 		exit_jaunt(cast_on)
 		return
 	enter_jaunt(cast_on)
+
+/mob/living/basic/creature/tiggles
+	name = "Miss Tiggles"
+
+/mob/living/basic/creature/hatchling
+	name = "hatchling"
+	health = 25
+	maxHealth = 25
+	health_scaling = FALSE
```

Comment:
```suggestion
	health_scaling = FALSE
	current_size = 0.85
```
I'm fixing an issue with initial current_size on #87021 by the way, if that's the reason why you're doing it on init.